app/config/services.yml

```diff
@@ -4,6 +4,20 @@ parameters:
     #parameter_name: value
 
 services:
-    #service_name:
-    #    class: AppBundle\Directory\ClassName
-    #    arguments: ['@another_service_name', 'plain_value', '%parameter_name%']
+    # default configuration for services in *this* file
+    _defaults:
+        # automatically injects dependencies in your services
+        autowire: true
+        # automatically registers your commands, form types, etc.
+        autoconfigure: true
+        public: false
+
+    # loads services from whatever directories you want (you can add directories!)
+    # this creates a service per class whose id is the fully-qualified class name
+    AppBundle\:
+        resource: '../../src/AppBundle/{Command,Form,EventSubscriber,Twig,Voter}'
```

Comment:
In the documentation, we place custom voters in a `Security` subnamespace instead of using `Voter` (see http://symfony.com/doc/current/security/voters.html).